apps/omg_watcher/lib/omg_watcher/api/status.ex

```diff
@@ -51,17 +55,20 @@ defmodule OMG.Watcher.API.Status do
   def get_status() do
     {:ok, eth_block_number} = EthereumHeight.get()
     {:ok, eth_block_timestamp} = Eth.get_block_timestamp_by_number(eth_block_number)
-    eth_syncing = Eth.syncing?()
+    eth_syncing = syncing?()
 
     validated_child_block_number = get_validated_child_block_number()
+    # wtf is eth diagnostics?
+    contracts = Eth.Diagnostics.get_child_chain_config()[:contract_addr]
```

Comment:
```suggestion
    contracts = OMG.Eth.Configuration.contracts()
```

Should work right? Then `Eth.Diagnostics.get_child_chain_config/0` can be dropped.

And `OMG.Eth.Diagnostics.get_node_diagnostics/0` moved into BlockQueue the only place where it's used.

Reply:
yeah, removed that and a couple of others